bugsnag-android-core/src/main/java/com/bugsnag/android/ImmutableConfig.kt

```diff
@@ -160,5 +161,38 @@ internal fun sanitiseConfiguration(
             configuration.buildUuid = buildUuid
         }
     }
+
+    // populate releaseStage
+    if (configuration.releaseStage == null) {
+        try {
+            val packageManager = appContext.packageManager
+            val ai = packageManager.getApplicationInfo(
+                packageName, PackageManager.GET_META_DATA
+            )
```

Comment:
How much overhead is there in doing this? Can the ApplicationInfo be shared with the logic to get the buildUuid above?

Reply:
There's minimal overhead in retrieving this information IIRC.